all/entrypoint.sh

```diff
@@ -60,6 +59,7 @@ DESCR_FILENAME="descriptor_set.pb"
 CSHARP_OPT=""
 SCALA_OPT=""
 OPENAPI_JSON=false
+DEPRICATION_MESSAGE="(deprecated. Please use --with-openapi-json-names)"
```

Comment:
What about something more specific? e.g.:
```suggestion
JSON_DEPRICATION_MESSAGE="(deprecated. Please use --with-openapi-json-names)"
```

Reply:
Also, typo... it should be `DEPRECATION`, not `DEPRICATION`

Reply:
thanks guys, I actually left it there by mistake, decided not to use it since the messages are slightly different when used.